src/uwtools/drivers/mpas.py

```diff
@@ -66,6 +66,23 @@ def namelist_file(self):
             schema=self.namelist_schema(),
         )
 
+    @tasks
+    def provisioned_rundir(self):
+        """
+        Run directory provisioned with all required content.
+        """
+        yield self.taskname("provisioned run directory")
+        required = [
+            self.files_copied(),
+            self.files_linked(),
+            self.namelist_file(),
+            self.runscript(),
+            self.streams_file(),
+        ]
+        if self.config["domain"] == "regional":
+            required.append(self.boundary_files())
+        yield required
+
```

Comment:
MPAS and MPASInit now have their own `provisioned_rundir` methods and the MPASBase class has only the interface definition.